src/app.coffee

```diff
@@ -36,15 +39,21 @@ defaults =
   driver: "memory"
   params: {}
   address: null,
-  
-if fs.existsSync(CONFIG_FILE)
-  try
-    config = JSON.parse(fs.readFileSync(CONFIG_FILE))
-  catch err
-    console.error err
-    process.exit 1
-else
-  config = {}
+  port: null
+
+if process.env.HOME
+  CONFIG_FILES.push path.join process.env.HOME, ".ofirehose.json"
```

Comment:
Did you test this? Something seems a little off about this CoffeeScript syntax but I don't even know because it's CoffeeScript

Reply:
yes, I test